crates/wasm/src/wasm_planner.rs

```diff
@@ -172,6 +191,12 @@ impl WasmPlanner {
     pub async fn plan(&mut self, refund_address: JsValue) -> WasmResult<JsValue> {
         utils::set_panic_hook();
 
+        // Calculate the gas that needs to be paid for the transaction based on the configured gas prices.
+        // Note that _paying the fee might incur an additional `Spend` action_, thus increasing the fee,
+        // so we slightly overpay here and then capture the excess as change later during `plan_with_spendable_and_votable_notes`.
```

Comment:
Is overpaying what is being done in plci as well?

Reply:
Yeah, it's just copied from pcli